test/e2e/scheduling/priorities.go

```diff
@@ -114,6 +114,7 @@ var _ = framework.KubeDescribe("SchedulerPriorities [Serial]", func() {
 
 		By("Create an RC with the same number of replicas as the schedualble nodes. One pod should be scheduled on each node.")
 		rc := createRC(ns, "scheduler-priority-selector-spreading", int32(len(nodeList.Items)), map[string]string{"name": "scheduler-priority-selector-spreading"}, f, podRequestedResource)
+		framework.WaitForRCToStabilize(cs, ns, rc.Name, framework.PodReadyBeforeTimeout)
```

Comment:
@k82cn Yes, we can, either can works well, but i'd like to get this in to see whether this can will fix the flaky.